patches/http-proxy+1.17.0.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/http-proxy/lib/http-proxy/passes/web-incoming.js b/node_modules/http-proxy/lib/http-proxy/passes/web-incoming.js
+index 995a0db..a5c37f5 100644
+--- a/node_modules/http-proxy/lib/http-proxy/passes/web-incoming.js
++++ b/node_modules/http-proxy/lib/http-proxy/passes/web-incoming.js
+@@ -50,7 +50,7 @@ module.exports = {
+    */
+ 
+   timeout: function timeout(req, res, options) {
+-    if(options.timeout) {
++    if(options.timeout || options.timeout === 0) {
```

Comment:
0 == false so we need to accept also exact 0 value

Reply:
Could we do this without patching if we set big enough timeout value?

Reply:
Yes that's possible too. Any idea about what's big enough?

Reply:
Few hours should be enough, see https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/ (–min-request-timeout).